004/index.html

```diff
@@ -64,13 +83,12 @@
 var d.value = links.payGrade;
 */
 
-var areas = [];
-
-links.forEach(function(link) {
+root.forEach(function(link) {
     areas.push(link.area);
 });
 
 areas = _.uniq(areas);
+areas.push("Choose a Department");
```

Comment:
prefer to have only data in the selector and initialize the view with a random department